peers/app_request_netrwork_metrics.go

```diff
@@ -0,0 +1,52 @@
+package peers
+
+import (
+	"errors"
+	"github.com/ava-labs/awm-relayer/config"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	ErrFailedToCreateAppRequestNetworkMetrics = errors.New("failed to create app request network metrics")
+)
+
+type AppRequestNetworkMetrics struct {
+	infoAPIBaseURL   string
+	pChainAPIBaseURL string
+
+	infoAPICallLatencyMS   *prometheus.GaugeVec
+	pChainAPICallLatencyMS *prometheus.GaugeVec
+}
+
+func NewAppRequestNetworkMetrics(cfg *config.Config, registerer prometheus.Registerer) (*AppRequestNetworkMetrics, error) {
+	infoAPICallLatencyMS := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
```

Comment:
Shall it be an histogram ?

Reply:
That's a good idea
i've made the changes.